scipy/sparse/csgraph/_laplacian.py

```diff
@@ -43,9 +48,28 @@ def laplacian(csgraph, normed=False, return_diag=False, use_out_degree=False):
     Notes
     -----
     The Laplacian matrix of a graph is sometimes referred to as the
-    "Kirchoff matrix" or the "admittance matrix", and is useful in many
+    "Kirchoff matrix" or just the "Laplacian", and is useful in many
     parts of spectral graph theory. In particular, the eigen-decomposition
-    of the laplacian matrix can give insight into many properties of the graph.
+    of the Laplacian can give insight into many properties of the graph, e.g.,
+    is commonly used for spectal data enmedding and clustering.
+
+    Unless the `inplace=True` parameter is used, the constructed Laplacian
+    doubles the memory use.
+
+    The Laplacian preserved the `dtype` and the format of the input
```

Comment:
I would talk about type. Or use double backticks.
```suggestion
    The Laplacian preserved the type and the format of the input
```